variants/heltec_vision_master_t190/platformio.ini

```diff
@@ -0,0 +1,13 @@
+[env:heltec-vision-master-t190]
+extends = esp32s3_base
+board = heltec_wifi_lora_32_V3
+build_flags = 
+  ${esp32s3_base.build_flags} 
+  -I variants/heltec_vision_master_t190
+  -D PRIVATE_HW  
+ ; -D PRIVATE_HW 
+lib_deps =
+  ${esp32s3_base.lib_deps}
+  lewisxhe/PCF8563_Library@^1.0.1
+  https://github.com/Bei-Ji-Quan/st7789
```

Comment:
This repo lib_dep and the other instances of it should be tied to a specific commit hash